### PR TITLE
One time submission

### LINKED ISSUE #15: Disallow multiple form submissions

### PR DESCRIPTION
**Pull request by lukasbischof**
Also implemented badges when you're logged in as an admin
image

closes #15

### CODE REVIEW

File: config/locales/de.yml [hunk 1]
```diff
@@ -109,3 +111,14 @@ de:
...
                 - Zeit
+  surveys:
+    index:
+      tooltips:
+        already_submitted: "Du hast diese Umfrage bereits ausgefüllt. Danke!"
+      admin_badges:
+        submission:
+          zero: "Noch keine Einreichungen"
+          one: "Eine Einreichung"
+          other: "%{count} Einreichungen"
```
Comment (TheFehr) [inline config/locales/de.yml]: This seems to not exist in any other translation

Comment (lukasbischof) [reply config/locales/de.yml]: Those are dynamic translations of Rails i18n framework for pluralization

Comment (lukasbischof) [reply config/locales/de.yml]: They gettin used in the bubbles of the list (see image attached to PR)

Comment (TheFehr) [reply config/locales/de.yml]: Ah so only admin, okay then it's fine

**Pull request merged**